src/components/players/list/index.js

```diff
@@ -12,61 +14,92 @@ class List extends React.Component {
     constructor(props) {
         super(props);
 
-        this.ref = createRef();
-
         this.state = {
-            selectedId: null,
-            selectedTeam: this.props.selectedTeam,
+            selectedPlayerId: null,
         };
+
+        this.ref = createRef();
     };
 
     componentDidUpdate(prevProps) {
-        if (prevProps.selectedTeam !== this.props.selectedTeam) {
-            this.setState({ selectedId: null });
+        if (prevProps.selectedTeam !== this.props.selectedTeam) { 
+            this.setState({ selectedPlayerId: null });
+        };
+
+        if (!prevProps.selectedTeam.isLoading && this.props.selectedTeam.isLoading) {
             this.ref.current.scrollTo(0, 0);
         };
     };
 
     renderPlayers() {
-        const { players, selectedTeam } = this.props;
-
-        return players.map((player, index) => {
-            const isSelected = this.state.selectedId === player.person_id;
-            const { teamColor } = player;
+        const { 
+            players, 
+            selectedTeam: {
+                isLoading,
+                team: { teamId },
+            }, 
+        } = this.props;
 
+        const emptyPlayers = new Array(20).fill({});
```

Comment:
Add todo comment above this:

`// TODO: Move this reset emptyPlayers into the players list reducer`

Reply:
Already fixed, resolving issue